changelogs/fragments/forward-error-correction-feature.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+- add support for setting FEC on interfaces
```

Comment:
```suggestion
- Add support for setting FEC on interfaces (https://github.com/ansible-collections/dellemc.os10/pull/78)
```

Reply:
My concern is that adding the URL for the pull request, creates a circular dependency. You must create a PR in github before you can create the changelog fragment, which makes things inconvenient.

Reply:
yeah, it's inconvenient. You could create an issue (which is preferable) that the PR fixes and put its URL in the fragment.

Reply:
Fine. I'll open an issue.

Reply:
If no response from the maintainers, I'll merge this tomorrow morning